pulp_smash/config.py

```diff
@@ -8,12 +8,15 @@
 that information.
 """
 import collections
+import itertools
 import json
 import os
 import warnings
 from copy import deepcopy
 from urllib.parse import urlparse
 
+import jsonschema
+import jsonschema.validators as jsonschema_validators
```

Comment:
That is stuff that remained from the first version. Before only jsonschema.validators was needed but as I wanted to have better error messages then I started importing the jsonchema module.

Thanks for noticing it, I've updated the PR and dropped that second import.